ai_edge_torch/generative/examples/stable_diffusion/diffusion.py

```diff
@@ -250,6 +250,7 @@ def __init__(self, config: unet_cfg.DiffusionModelConfig):
 
     attention_config = layers_cfg.AttentionConfig(
         num_heads=config.transformer_num_attention_heads,
+        head_dim=block_out_channels[0] // config.transformer_num_attention_heads,
```

Comment:
Attention dims are different for each block. Should construct attention config in the down_encoders loop below. Check `AttentionBlock2DConfig` and `CrossAttentionBlock2DConfig`.

Also, the attention configs in mid_block and up_decoders should be fixed as well.